iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/AbstractOptions.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * @author Chamseddine BENHAMED <chamseddine.benhamed at rte-france.com>
+ */
+public abstract class AbstractOptions<T> {
+    protected Optional<Set<String>> extensions = Optional.empty();
```

Comment:
Optional as a field in considered as a bad design, use Set<String> that can be null inside AbstractOptions class and Optional<Set<String>> in getter.
[https://blog.joda.org/2014/11/optional-in-java-se-8.html](https://blog.joda.org/2014/11/optional-in-java-se-8.html)

Reply:
Done